### PR TITLE
Fix code style

### PR DESCRIPTION
**Pull request by kjanosz**

### CODE REVIEW

File: scalameter-core/src/main/scala/org/scalameter/picklers/Pickler.scala [hunk 1]
```diff
@@ -12,7 +12,7 @@ abstract class Pickler[T] extends Serializable {
 
   final def unpickle(a: Array[Byte]): T = {
     val (obj, newPos) = unpickle(a, 0)
-    if (newPos > 0) sys.error(s"Malformed data. Input: ${a.toVector}. Remaining: ${a.toVector.slice(newPos, a.length)}, $newPos")
+    if (newPos > 0) sys.error(s"Malformed data. Input: ${a.toVector}. Remaining: ${a.toVector.slice(newPos, a.length)}.")
```
Comment (axel22) [inline scalameter-core/src/main/scala/org/scalameter/picklers/Pickler.scala]: Please use `mkString` here, and delimit the elements with commas.

File: scalameter-core/src/main/scala/org/scalameter/picklers/containers.scala [hunk 2]
```diff
@@ -3,6 +3,7 @@ package org.scalameter.picklers
 import language.higherKinds
 
 import java.util.Date
+import scala.annotation.tailrec
 import scala.collection.generic.CanBuildFrom
 import Implicits._
```
Comment (axel22) [inline scalameter-core/src/main/scala/org/scalameter/picklers/containers.scala]: Please add a full import name here.

File: scalameter-core/src/main/scala/org/scalameter/picklers/containers.scala [hunk 3]
```diff
@@ -24,17 +25,17 @@ abstract class TraversablePickler[C[_] <: Traversable[_], T: Pickler] extends Pi
     val pickler = implicitly[Pickler[T]]
     val builder = canBuildFrom()
 
-    @annotation.tailrec
-    def _unpickle(times: Int, from: Int): (C[T], Int) = {
+    @tailrec
+    def unpickle(times: Int, from: Int): (C[T], Int) = {
```
Comment (axel22) [inline scalameter-core/src/main/scala/org/scalameter/picklers/containers.scala]: Awesome, thanks!

**Comment by axel22** [comment]
Thanks! LGTM after minor comments are addressed.

**Comment by kjanosz** [comment]
@axel22: Fixed.

File: scalameter-core/src/main/scala/org/scalameter/picklers/Pickler.scala (hunk 1 above)
Comment (kjanosz) [inline scalameter-core/src/main/scala/org/scalameter/picklers/Pickler.scala]: I've used `mkString` with `start` and `end` arguments (set to "[" and "]").

**Pull request merged**